libs/langchain/tests/unit_tests/test_keybert_link_extractor.py

```diff
@@ -1,5 +1,5 @@
-from ragstack_langchain.graph_store.extractors import KeybertLinkExtractor
-from ragstack_langchain.graph_store.links import Link
+from langchain_community.graph_vectorstores.extractors import KeybertLinkExtractor
```

Comment:
We can probably drop most/all of these tests:

1. Graph Store functionality is tested in #596.
2. Extractor functionality is tested in langchain.

Reply:
LangChain only runs the unit tests, not the integration ones.
So I removed the ones that are tested in LC's unit tests but kept the ones in integration tests.